library/include/ck/library/tensor_operation_instance/gpu/batched_gemm_masking_scale_softmax_gemm_permute.hpp

```diff
@@ -36,7 +36,24 @@ void add_device_batched_gemm_masking_scale_softmax_gemm_permute_xdl_cshuffle_f16
                                                                     PassThrough,
                                                                     Scale,
                                                                     PassThrough,
-                                                                    PassThrough>>>& instances);
+                                                                    PassThrough,
+                                                                    true>>>& instances);
+
+void add_device_batched_gemm_scale_softmax_gemm_permute_xdl_cshuffle_f16_f16_f16_f16_gmk_gnk_gno_gmo_instances(
```

Comment:
As non-masking instances are added, can we rename the file?

Reply:
`masking` is removed from file name. And since scaling op is enabled for all instances, `scale` is removed from file names and class names too. We are left with just `gemm_softmax_gemm_permute`